hll/test/HllSketchTest.cpp

```diff
@@ -162,27 +162,23 @@ TEST_CASE("hll sketch: check ser sizes", "[hll_sketch]") {
 }
 
 TEST_CASE("hll sketch: exercise to string", "[hll_sketch]") {
-  test_allocator_total_bytes = 0;
-  {
-    hll_sketch_test_alloc sk(15, HLL_4);
-    for (int i = 0; i < 25; ++i) { sk.update(i); }
-    std::ostringstream oss(std::ios::binary);
-    oss << sk.to_string(false, true, true, true);
-    for (int i = 25; i < (1 << 20); ++i) { sk.update(i); }
-    oss << sk.to_string(false, true, true, true);
-    oss << sk.to_string(false, true, true, false);
-
-    sk = hll_sketch_test_alloc(8, HLL_8);
-    for (int i = 0; i < 25; ++i) { sk.update(i); }
-    oss << sk.to_string(false, true, true, true);
-  }
-  REQUIRE(test_allocator_total_bytes == 0);
+  hll_sketch sk(15, HLL_4);
```

Comment:
Seems reasonable to have some non-test_allocator usage, too, but why was it explicitly removed in this case?

Reply:
Yes, I thought about splitting this set of tests into simple tests that can serve as usage examples, and more elaborate allocation tests with this additional clutter (passing allocators, checking allocated bytes and so on). I am afraid it is a bit more involved than I planned at this moment.

Reply:
Fair. Gigantic PRs are obnoxious in many ways and we don't need them all to do everything. Thanks.